docs/section21/Rule21-6.md

```diff
@@ -0,0 +1,40 @@
+
+# Boiler - Rule 21-6  
+
+**Rule ID:** 21-6  
+**Rule Description:** When baseline building includes two boilers each shall stage as required by load.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12
+2. B-RMR is not modeled with purchased heating.
+3. B-RMR is modeled with two boilers.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12: `if PLACEHOLDER_AIRSIDE_SYSTEM_RULE-X-XX:`
+2. B-RMR is not modeled with purchased heating: `if NOT Rule-21-1:`
+3. B-RMR is modeled with two boilers: `if ( Rule-21-5 ) AND ( LEN(B_RMR.ASHRAE229.boilers) == 2)`
+
```

Comment:
does this need to say 21-5 is pass?

Reply:
Updated.

Reply:
Per above, I think lines #24 and #25 can be removed.

Reply:
I added "Applicability Diagram" to the rule spreadsheet.

Reply:
Let's discuss on the call

Reply:
Let's discuss on the call